index.d.ts

```diff
@@ -94,6 +94,11 @@ declare namespace captureWebsite {
 		*/
 		waitForElement?: string;
 
+		/**
+		An object which specifies clipping region of the page.
```

Comment:
No point in mentioning the type. That's already clear from the actual type defs. Instead, elaborate on what a clipping region is.